.github/workflows/ci-build.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [3.6, 3.7, 3.9]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Has NeuroSpin updated its default version of Python? The only reason we were still supporting v3.6 was because it was the default on most workstations.

Reply:
Python 3.6 (EOL last year) was installed on ubuntu 18.04 machines, (which is reaching EOL in the next semester) , new workstations are shipped with 20.04 nowadays, and [pyenv](https://github.com/pyenv/pyenv) can still be installed to have the desired  version. 
However, I might recon that I have been a bit greedy. Tensorflow (one of the supported backend) is only supported on 3.7, see the [compatibility matrix](https://www.tensorflow.org/install/source#tested_build_configurations). 

With the supported python version , comes also the question of the numpy version to. I guess relying on [oldest-supported-numpy](https://github.com/scipy/oldest-supported-numpy) could  be a viable solution to avoid the dependency nightmare.

Reply:
Furthermore, this only the tested python versions. The actual supported version are described in `setup.py` where I also made the change.